patroni/utils.py

```diff
@@ -977,3 +990,33 @@ def unquote(string: str) -> str:
     except ValueError:
         ret = string
     return ret
+
+
+def get_major_version(bin_dir: Optional[str] = None, bin_name: str = 'postgres') -> str:
+    """Get the major version of PostgreSQL.
+
+    It is based on the output of ``postgres --version``.
+
+    :param bin_dir: path to PostgreSQL binaries directory. If ``None`` it will use the first ``postgres`` binary that
```

Comment:
```suggestion
    :param bin_dir: path to PostgreSQL binaries directory. If ``None`` or empty string it will use the first *bin_name* binary that
```